Dockerfile

```diff
@@ -1,19 +1,10 @@
-FROM gradle:5.5-jdk8 as builder
+FROM node:12.16.3
```

Comment:
not using an alpine image here because mongodb doesn't have a prebuilt binary for alpine